theme/themes/pxt/globals/site.variables

```diff
@@ -392,4 +392,7 @@
 /*---------------------------
    Tutorial Code Validation
 ----------------------------*/
-@tutorialCodeValidationMoveOn: @green;
+@tutorialCodeValidationCorrectBackground: @green;
+@tutorialCodeValidationCorrectTextColor: @white;
+@tutorialCodeValidationCorrectBackgroundMicrobit: #cd0365;
+@tutorialCodeValidationCorrectTextColorMicrobit: @white;
```

Comment:
we shouldn't need two variables for this; should just override the two above in target